types/runtime/index.d.ts

```diff
@@ -20,49 +21,51 @@ export interface Loader {
 
 export class View {
   constructor(runtime: Runtime, config?: any);
-  initialize(dom?: Element | string): View;
+  initialize(dom?: Element | string): this;
   finalize(): void;
-  logLevel(level: number): View;
-  renderer(renderer: 'canvas' | 'svg' | 'none'): View;
-  loader(loader: Loader): View;
-
-  hover(): View;
-  run(): View;
-  runAfter(callback: (view: View) => void, enqueue?: boolean, priority?: number): View;
+  logLevel(level: number): this;
+  _redraw: boolean;
```

Comment:
I know you would like to have these but I don't want `_*` in the official typings. You can always say `(renderer as any)._redraw` in your app.